ccd-definition/ComplexTypes/ResponseDocumentFile.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "ID": "ResponseDocumentFile",
+    "ListElementCode": "responseDocument",
```

Comment:
can this just be file and the complex type be ResponseDocument? Otherwise we have ResponseDocumentFile.responseDocument. ResponseDocument.file seems nicer imo

Reply:
done